maro/simulator/scenarios/vm_scheduling/topologies/azure.2019.10k/config.yml

```diff
@@ -11,17 +11,36 @@ CPU_READINGS: "~/.maro/data/vm_scheduling/.build/azure.2019.10k/vm_cpu_readings-
 
 PROCESSED_DATA_URL: "https://marodatasource.blob.core.windows.net/vm-scheduling-azure/azure.2019.10k/azure.2019.10k.tar.gz"
 
+# True means kill all VMs on the overload PM.
+# False means only count these VMs as failed allocation, but not kill them.
+KILL_ALL_VMS_IF_OVERLOAD: True
+
+# Oversubscription configuration.
+# Max CPU oversubscription rate.
+MAX_CPU_OVERSUBSCRIPTION_RATE: 1.15
+# Max memory oversubscription rate.
+MAX_MEM_OVERSUBSCRIPTION_RATE: 1
+# Max CPU utilization rate.
+MAX_UTILIZATION_RATE: 1
+
 PM:
-  AMOUNT: 100
-  CPU: 32
-  # The unit of the memory is GB.
-  MEMORY: 128
-  # GPU: 4
-  # NOTE: Energy consumption parameters should refer to more research.
-  POWER_CURVE:
-    # The calibration parameter used for the CPU utilization vs energy consumption model.
-    CALIBRATION_PARAMETER: 1.4
-    # The idle power usage of a machine.
-    BUSY_POWER: 10
-    # The busy power usage of a machine.
-    IDLE_POWER: 1
+  - SKU_type: 0
+    amount: 50
+    CPU: 32
+  # GPU: 0
+    memory: 128
+    price: 0
```

Comment:
set default price > 0

Reply:
Should we implement `price` in this PR? Including pricing model?

Reply:
no need to implement pricing model in this PR